libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -25,7 +25,7 @@ use std::time::Instant;
 use uuid::Uuid;
 
 pub use error::{AuthorizerError, ConnectionManagerError};
-pub use notification::{ConnectionManagerNotification, NotificationIter};
+pub use notification::ConnectionManagerNotification;
```

Comment:
Commit message still says it removes connection-manager-notification-iter-try-next  but that has already been removed